src/components/Navbar/Navbar.scss

```diff
@@ -22,13 +22,21 @@ ul {
 }
 
 .content {
-  background-color: $sixth-color;
+  // background-color: $sixth-color;
   border-radius: $spacing-xsmall;
   min-height: 95vh;
   margin-top: $spacing-small;
   padding: $spacing-small;
 }
 
+.dark-mode-bg {
+  background-color: black;
+}
+
+.light-mode-bg {
+  background-color: $sixth-color;
+}
+
 .navbar {
```

Comment:
You can use sass nested feature to accomplish this. Refactor the code in this way:
```scss
.box {
  /* base css properties with the light mode values */
  &.dark {
    /* override light mode properties */
  }
}
```

Reply:
Done.